packages/app/src/systems/Chains/eth/hooks/useBlock.tsx

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@tanstack/react-query';
+import { formatDistanceToNow } from 'date-fns';
+
+import { useEthAccountConnection } from './useEthAccountConnection';
+
+function calculateBlockAge(timestamp?: number) {
+  if (!timestamp) {
+    return 'N/A';
+  }
+  const blockDate = new Date(timestamp * 1000);
+  const diffInDays = formatDistanceToNow(blockDate, { addSuffix: true });
+  return diffInDays;
+}
```

Comment:
we should create a `date.ts` file in `systems/Core/utils` and put this function into there
suggestion for name: `calculateDateDiff(timestamp?: number)`